migrations/20220516100946_orders.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async (knex) => {
+  const [extInstalled] = await knex('pg_extension').select('*').where({ extname: 'uuid-ossp' })
+
+  if (!extInstalled) {
+    await knex.raw('CREATE EXTENSION "uuid-ossp"')
+  }
```

Comment:
shouldn't need this check also. It's in the first migration as some cloud envs install the extension automatically whilst others don't